### PR TITLE
Delete unused variable

### PR DESCRIPTION
**Pull request by abergmeier**
For now this is not used so just ~~uncomment~~ delete so nobody searches for that.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ function main {
 
       file_dirname=$(dirname "${file}")
       file_basename=$(basename "${file}")
-      file_name="${file_basename%.*}"
+      #file_name="${file_basename%.*}"
```
Comment (neenjaw) [inline entrypoint.sh]: How does this differ from what you're doing in #22 to get a basename to create a brand new file?

Comment (abergmeier) [reply entrypoint.sh]: This is just cleanup so nobody wonders where `file_name` gets used.

Comment (neenjaw) [reply entrypoint.sh]: Let's just delete it then rather than comment. It exists in the git history.

Comment (abergmeier) [reply entrypoint.sh]: Done.

**Pull request merged**